xdsl/dialects/onnx.py

```diff
@@ -673,6 +675,82 @@ def verify_(self) -> None:
                 f"Only one value attribute must be provided, but {used_attrs} were specified"
             )
 
+    def print(self, printer: Printer):
+        if self.value:
+            printer.print_string(" ")
+            printer.print_attribute(self.value)
+        printer.print_string(" ")
+        printer.print_string("{")
+        if self.value_float:
+            printer.print_string("value_float")
+            printer.print_string(" = ")
+            printer.print_attribute(self.value_float)
+        elif self.value_floats:
+            pass
+        elif self.value_int:
+            printer.print_string("value_int")
+            printer.print_string(" = ")
+            printer.print_attribute(self.value_int)
+        elif self.value_ints:
+            printer.print_string("value_ints")
+            printer.print_string(" = ")
+            arr: list[int] = [value.value.data for value in self.value_ints]
+            printer.print(arr)
+        elif self.value_string:
+            pass
+        elif self.value_strings:
+            pass
```

Comment:
I would recommend adding a helper here, one that will give the only value for this instance, like `self.get_value()`